.travis.yml

```diff
@@ -6,28 +6,42 @@ branches:
 
 env:
   global:
+    - OPENPMD_GIT_REF="0.11.1-alpha"
+
     # (1,2) Skip building for Python 2.7 on all platforms
-    - CIBW_SKIP="cp27-* pp27-*"
+    # (3) GNU 7.3.1 & MPark.Variant 1.4.0 on i686:
+    #     https://github.com/pypa/manylinux/issues/543
+    #       variant.hpp:2241:9: internal compiler error:
+    #       unexpected expression ‘I’ of kind template_parm_index
+    #       typename T = lib::type_pack_element_t<I, Ts...>,
+    - CIBW_SKIP="cp27-* pp27-* *-manylinux_i686"
     # Install dependencies on Linux and OSX
     - CIBW_BEFORE_BUILD="bash -x .github/library_builders.sh"
 
 jobs:
   include:
     # perform a linux ARMv8 build
-    # blocked by https://github.com/ornladios/ADIOS2/issues/2137
+    # blocked by https://github.com/scikit-build/cmake-python-distributions/issues/96
+    #            https://github.com/pypa/manylinux/issues/544
     # - services: docker
-    #   arch: aarch64
```

Comment:
upstream docs typo: this is called `arm64` on Travis-CI
https://github.com/joerick/cibuildwheel/pull/322